app/controllers/application_controller.rb

```diff
@@ -4,8 +4,17 @@ class ApplicationController < ActionController::Base
     protected
   
     def configure_permitted_parameters
-      devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
-      devise_parameter_sanitizer.permit(:account_update, keys: [:name])
+      if resource_class == Student
+        devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
+        devise_parameter_sanitizer.permit(:account_update, keys: [:name])
+      end 
+    end
+    
+    def configure_permitted_parameters
```

Comment:
there are two duplicated methods with name `configure_permitted_parameters`, that is wrong